### PR TITLE
Feature/raw mini

### PR DESCRIPTION
**Pull request by jabbenseth**
retry for #752

### CODE REVIEW

**Review by fmessmer — CHANGES_REQUESTED** [review]
you need to add `raw-mini` to `cob_supported_robots` in order to get checks activated...
please also add a tmp commit with the overlays from https://github.com/ipa320/cob_common/pull/281 and `cob_supported_robots` so we can verify tests pass....once they pass, the commit can be removed again...

File: cob_bringup/components/raw_mini_base.launch [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+  <group ns="base">
+    <group unless="$(arg sim)">
+      <node pkg="rosserial_python" type="serial_node.py" name ="rosserial" output="screen">
+        <param name="port" value="/dev/ttyACM0"/>
+        <param name="baud" value="115200"/>
+      </node>
+
+      <node pkg="cob_mecanum_controller" type="cob_mecanum_controller_node" name="mecanum_controller" output="log">
```
Comment (fmessmer) [inline cob_bringup/components/raw_mini_base.launch:16]: add missing dependency

File: cob_bringup/robots/raw-mini.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<launch>
+
+  <!-- args -->
```
Comment (fmessmer) [inline cob_bringup/robots/raw-mini.xml:4]: no machine tags needed?
is there just one pc on raw_mini?

File: cob_hardware_config/robots/raw-mini/config/base_laser_filter.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+scan_filter_chain:
```
Comment (fmessmer) [inline cob_hardware_config/robots/raw-mini/config/base_laser_filter.yaml]: doesn't seem to be used - remove?

File: cob_hardware_config/robots/raw-mini/config/footprint_observer_params.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# node from which initial footprint is read
```
Comment (fmessmer) [inline cob_hardware_config/robots/raw-mini/config/footprint_observer_params.yaml]: doesn't seem to be used - remove?

File: cob_hardware_config/robots/raw-mini/urdf/properties.urdf.xacro [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
```
Comment (fmessmer) [inline cob_hardware_config/robots/raw-mini/urdf/properties.urdf.xacro:2]: do you want to prepare the `cob_calibration_data` structure?

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_bringup/robots/raw-mini.xml (hunk 2 above)
Comment (jabbenseth) [reply cob_bringup/robots/raw-mini.xml:4]: yes and yes (maybe I should post a picture somewhere) raw-minis footprint is around the size of a DIN-A4 Paper --> no space for more than one pc

File: cob_hardware_config/robots/raw-mini/urdf/properties.urdf.xacro (hunk 5 above)
Comment (jabbenseth) [reply cob_hardware_config/robots/raw-mini/urdf/properties.urdf.xacro:2]: on the robot there is not much to calibrate --> I think it is not needed and will not be needed in the future --> no (if that is ok with you)

**Comment by fmessmer** [comment]
I rebased and added a roslaunch check....I merged cob_control and cob_common...let's see whether this passes...

**Review by fmessmer — APPROVED** [review]

**Pull request merged**